config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    mongoUri: process.env.MONGO_URI
```

Comment:
Can you set this to MONGODB_URI?  That's the variable that Heroku injects automatically.  I could change it on the Heroku side but I'm not sure if there are any side-effects if we move away from the default that Heroku automatically generates.